src/test/resources/application.properties

```
@@ -4,5 +4,6 @@ spring.datasource.url=jdbc:h2:mem:testdb
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
+stripe.api.key=invalid_Stripe_api_key
```

```suggestion

stripe.api.key=${STRIPE_API_KEY}
```
And add this field to `.env.sample`.